cinema/serializers.py

```diff
@@ -0,0 +1,18 @@
+from rest_framework import serializers
+from .models import Movie
+
+class MovieSerializer(serializers.Serializer):
+    id = serializers.IntegerField(read_only=True)
+    title = serializers.CharField(required=True)

```

Comment:
The `title` field is marked as `required=True`, which is the default behavior for `CharField`. You can omit this parameter unless you want to explicitly specify it.